src/reporting/clients/IEC3ConfigurationsClient.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import { EC3Configuration, EC3ConfigurationCreate, EC3ConfigurationUpdate } from "../interfaces/EC3Configurations";
+import type { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IEC3ConfigurationsClient {
+  /**
+   * Gets all EC3 Configurations within the context of a Project. This method returns the full list of EC3 Configurations.
+   * @param {string} projectId The project Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`.
+   * @param {number} top The number of entities to load per page.
+   * @memberof IEC3ConfigurationsClient
+   * @link https://developer.bentley.com/apis/carbon-calculation/operations/get-ec3-configurations/
+   */
+  getConfigurations(
+    accessToken: AccessToken,
+    projectId: string,
+    top?: number
+  ): Promise<EC3Configuration[]>;
+
+  /**
+   * Gets an async paged iterator for EC3 Configurations within the context of a Project.
+   * This method returns an iterator which loads pages of mappings as it is being iterated over.
```

Comment:
The pages are most likely not `of mappings`.

Reply:
Triple checked documentation